pkg/network/tracer/tracer_test.go

```diff
@@ -1539,22 +1545,55 @@ func TestHTTPSViaOpenSSLIntegration(t *testing.T) {
 		t.Skip("this feature is not yet support on arm")
 	}
 
-	wget, err := exec.LookPath("wget")
-	if err != nil {
-		t.Skip("wget not found; skipping test.")
+	tlsLibs := []*regexp.Regexp{
+		regexp.MustCompile(`/[^\ ]+libssl.so[^\ ]*`),
+		regexp.MustCompile(`/[^\ ]+libgnutls.so[^\ ]*`),
 	}
-
-	ldd, err := exec.LookPath("ldd")
-	if err != nil {
-		t.Skip("ldd not found; skipping test.")
+	tests := []struct {
+		name     string
+		fetchCmd []string
+	}{
+		{name: "wget", fetchCmd: []string{"wget", "--no-check-certificate", "-O/dev/null"}},
+		{name: "curl", fetchCmd: []string{"curl", "--http1.1", "-k", "-o/dev/null"}},
```

Comment:
Thank you for doing this 🙇

Reply:
gnutls support need this fix : https://github.com/DataDog/datadog-agent/pull/9981/commits/999f9aa9fb0f873045ecedafab27c56fa9b8e1c0